gateway/manager/resharder.ts

```diff
@@ -197,6 +197,7 @@ export async function tellWorkerToPrepare(resharder: Resharder, shardId: number)
 
       if (messageData.t === "READY") {
         const payload = messageData.d as DiscordReady;
+        shard.resumeGatewayUrl = payload.resume_gateway_url;
```

Comment:
I think this should also be done in the handleMessage wherever the READY payload is being handled

Reply:
sorry what
there's only one place in the handleMessage where the READY payload is being handled, which is right here

Reply:
OH YOU MEAN THE gateway/shard/handleMessage.ts file